Update the matrix link to join the space

altf4arnold opened the pull request:
Clicking on the link in the current state will ask to log in with a username hosted on chat.gnuradio.org. Additional manipulation is required to add it to a matrix client (when a person already has one set).

The new version of the link will propose to either create an account/install an app from scratch or will just add itself to the already existant matrix client of the user

marcusmueller commented on content/contact/_index.md:
```diff
@@ -7,7 +7,7 @@ title = "Contact Us"
 
 * via Email using the GNU Radio discussion mailing list `discuss-gnuradio@gnu.org`, to which you can sign up [here](https://lists.gnu.org/mailman/listinfo/discuss-gnuradio),
-* via Matrix on our Matrix homeserver, e.g. using [chat.gnuradio.org](https://chat.gnuradio.org), or
+* via Matrix on our Matrix homeserver, e.g. using [chat.gnuradio.org](https://matrix.to/#/#gnuradio-community:gnuradio.org), or
```
Now the link text says "using chat.gnuradio.org", which means "using the element client hosted on chat.gnuradio.org", but the link target is just a generic matrix.to link, bringing users through complicated client choice instead of the chat.gnuradio.org client. (we really want everyone to join as easily as possible, agreed, but the matrix.to "oh here's a bunch of clients you might choose from" is really counter-productive there, if a user has IT problems already. We do have existing user feedback on that.)

So, don't think that's what we intended here, I'm afraid.

altf4arnold replied:
Oh. Do you think that having a "hybrid" solution could work? (like, a signup (joining on chat.gnuradio.org) or join link (joining to matrix.to)) ?

Or just beeing more verbose in the line that speaks about matrix?

marcusmueller commented:
Note that I think it's the right approach to drop people directly to the space, please make that change, just the detour through matrix.to is negative :)

altf4arnold committed:
Deprecation message at build

ERROR deprecated: .Site.DisqusShortname was deprecated in Hugo v0.120.0 and will be removed in Hugo 0.137.0. Use .Site.Config.Services.Disqus.Shortname instead.

altf4arnold committed:
Links are not distinguishable from the other content

altf4arnold commented:
Ok. So apparently, the issue is a little bit deeper. Turns out that the link on the website was already good enough. It just wasn't visible (unless  you know it is there). 

So instead of changing the links, I changed the CSS to make it stand out more.

I also got a deprecation error at build time. So I also changed that
